### PR TITLE
DDBTEAM-464: Added command to remove local uploaded files

### PR DESCRIPTION
**Pull request by cableman**
Symfony command to remove cover file form local storage if they have been uploaded

https://jira.itkdev.dk/browse/DDBTEAM-464

### CODE REVIEW

**Comment by cableman** [comment]
The error from checks are `Line exceeds 120 characters` in constructers.

File: src/Command/CleanUpCommand.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    private $coverRepository;
+    private $coverStoreService;
+    private $entityManager;
+
+    protected static $defaultName = 'app:image:cleanup';
+
+    /**
+     * CleanUpCommand constructor.
+     */
+    public function __construct(CoverRepository $coverRepository, CoverStoreService $coverStoreService, EntityManagerInterface $entityManager)
```
Comment (tuj) [inline src/Command/CleanUpCommand.php:31]: Consider adding an ignore rule to phpcs regarding line length.

**Review by tuj — APPROVED** [review]
Approved with comment

File: src/Command/CleanUpCommand.php (hunk 1 above)
Comment (tuj) [reply src/Command/CleanUpCommand.php:31]: https://github.com/danskernesdigitalebibliotek/ddb-cover-service/blob/develop/phpcs.xml.dist#L18

Comment (cableman) [reply src/Command/CleanUpCommand.php:31]: Added :-)

**Pull request merged**